packages/@biomejs/biome/README.pt-br.md

```diff
@@ -0,0 +1,124 @@
+<p align="center">
+    <img alt="Biome - Cadeia de ferramentas da web"
```

Comment:
```suggestion
    <img alt="Biome - Conjunto de ferramentas da web"
```
Previously we translated this into "Conjunto de ferramentas", so would be good to keep the same here